README.md

```diff
@@ -1,11 +1,12 @@
 # SSP Operator
-Operator that manages Scheduling, Scale and Performance addons for [KubeVirt](https://kubevirt.io)
+Operator that installs resources for [KubeVirt](https://kubevirt.io)
```

Comment:
I think by this change, users won't know what `SSP` means. Since SSP means `Scheduling, Scale and Performance`.

Reply:
I agree with the change. The deployed resources are not only related to scheduling, scale and performance anymore but we decided to stick with SSP's name. However, I would phrase it differently, maybe `deploys additional resources for KV` instead?

Reply:
You're right, but I disagree keeping then the same name for the operator, as it would confuse users who will ask "what is ssp-operator?" or "what does ssp-operator means?" (word "ssp" itself).

Reply:
While I agree with that we still decided to keep the name for now because changing it would require a lot of extra changes, e.g. in HCO, image names, etc.

Reply:
How about a new paragraph explaining what the acronym ssp stands for? (indicating the original and current purpose)

Reply:
@0xFelix Right, what about `Software Solution Provider (SSP)`? since this operator provides multiple KubeVirt software solutions, such as Template Validator, InstancesTypes, VM Console Proxy, and soon will be there Tekton Tasks.

Reply:
Let's keep it as `Operator that deploys additional KubeVirt resources`. I'll also send PR to update repository about summary.

Reply:
I've sent PR to update it in https://github.com/kubevirt/project-infra/pull/2778.